packages/cli/src/commands/account/register-metadata.ts

```diff
@@ -17,21 +20,34 @@ export default class RegisterMetadata extends BaseCommand {
       required: true,
       description: 'The url to the metadata you want to register',
     }),
+    force: flags.boolean({ description: 'Ignore checks' }),
   }
 
   static examples = [
     'register-metadata --url https://www.mywebsite.com/celo-metadata --from 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95',
   ]
 
   async run() {
-    const { flags } = this.parse(RegisterMetadata)
-    this.kit.defaultAccount = flags.from
+    const res = this.parse(RegisterMetadata)
+    this.kit.defaultAccount = res.flags.from
 
     await newCheckBuilder(this)
-      .isAccount(flags.from)
+      .isAccount(res.flags.from)
       .runChecks()
 
-    const accounts = await this.kit.contracts.getAccounts()
-    await displaySendTx('registerMetadata', accounts.setMetadataURL(flags.url))
+    const metadataURL = res.flags.url
+
+    try {
+      if (!res.flags.force) {
+        const metadata = await IdentityMetadataWrapper.fetchFromURL(metadataURL)
+        console.info('Metadata contains the following claims: \n')
+        await displayMetadata(metadata, this.kit)
+        console.info()
+      }
+      const accounts = await this.kit.contracts.getAccounts()
+      await displaySendTx('registerMetadata', accounts.setMetadataURL(metadataURL))
+    } catch (error) {
+      console.error(`Cannot set metadata URL ${metadataURL}: ${error.toString()}`)
+    }
```

Comment:
I would recommend keeping this `try` block smaller. Maybe something like:
```suggestion
   if (!res.flags.force) {
     try {
        const metadata = await IdentityMetadataWrapper.fetchFromURL(metadataURL)
        console.info('Metadata contains the following claims: \n')
        await displayMetadata(metadata, this.kit)
        console.info() // Print a newline.
      } catch (error) {
        console.error(`Metadata could not be retrieved from ${metadataURL}: ${error.toString()}`)
      }
    }
   
    const accounts = await this.kit.contracts.getAccounts()
    await displaySendTx('registerMetadata', accounts.setMetadataURL(metadataURL))
```